### DISCUSSION TITLE
Anyone using this to sync files for Logseq?

### DISCUSSION THREAD

**Discussion by seanleblanc** [MobiusSync/MobiusSync#159] [Q&A]
I set up a logseq folder to sync with Mobius Sync. I see the files show up via the Files application. However, when I point the LogSeq application to this folder, it seemingly complains about file permissions? ("No permission, please clear cache and re-open graph folder")

Anyone else using Mobius Sync to sync up files for logseq and what did you have to do to get it to work?

**Comment by jmhammond** [comment]
I’m not (yet), but this post suggests a solution https://discuss.logseq.com/t/please-choose-valid-directory-issue-on-iphone/11568/7
